src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = function(firstNumber) {
+    count++;
+
+    return function(secondNumber) {
+      if (count > 3) {
+        return (count - 3) % 2 === 0
+          ? firstNumber + secondNumber
```

Comment:
we have to decide where do you want to return the sum, either here or outside of if statement, having both exlude themselves